.github/workflows/Tests.yaml

```diff
@@ -216,7 +216,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        compiler: [gcc-7, gcc-8, gcc-9, clang-5.0, clang-8, clang-9]
+        compiler: [gcc-7,gcc-8,gcc-9,clang-6.0,clang-8,clang-9,clang-10]
```

Comment:
Do we need to keep clang-6 support around for a particular supercomputer?

Reply:
No, not a supercomputer, but we support clang-5 and newer.

Reply:
I suppose my question is: Do you think we can drop any further clang versions?

Reply:
I don't see a reason to do that. Ubuntu 18.04 ships clang-6 and two years isn't that old :)